cloud_edition/flexibility_mode/docs/crontasks.rst

```diff
@@ -0,0 +1,56 @@
+Crontasks settings
+==================
+
+You find here, some good practices about crontasks management in Akeneo PIM Flexibility Edition :
+
+1. CRONs we provide by default uses a custom wrapper script defined by *SHELL* that call the PIM "bin/console" command.
+
+   It avoid us to write on each cron line the full script path to call and the redirection of logs.
+
+   Adding your own cron lines failes because it tries to run your command with our wrapper.
+   
+   To run your own script normally (without our wrapper), you just have to redefine the shell variable like this "SHELL=/bin/bash" before them
+    
+2. We would like to remind you that all servers are using UTC time (and so on for CRON).
+   
+   Think to convert you desired local time to UTC time.
```

Comment:
Think about converting

(I don't understand the sentence personally)

Reply:
So make sure to always make the conversion to UTC time

Reply:
Remember to convert your desired ...